webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.js

```diff
@@ -10,7 +10,7 @@ const AlertBody = ({
 
     {title && <strong>{title}</strong>}
 
-    <span dangerouslySetInnerHTML={{ __html: message }} />
+    {message}
```

Comment:
i believe this was here to allow rails to send formatted html? would that still work?

Reply:
I haven't found any code doing that.

@tbrisker showed me this issue https://projects.theforeman.org/issues/24807 and we see it problematic in some way.

We decided that once we will find those places in the UI we will fill a new bug and fix them.

Reply:
The helpers we use actually escape any HTML already. If any notification is found sending HTML we should fix it.